tests/duck.spec.js

```diff
@@ -0,0 +1,41 @@
+const { test, expect } = require('@playwright/test');
+
+test('basic test', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
+  const title = page.locator('.navbar__inner .navbar__title');
+  await expect(title).toHaveText('Playwright');
+});
+
+test('duck go checking logo', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const logo = page.locator('#logo_homepage_link');
+  await expect(logo).toBeVisible();
+});
+
+test('T2', async ({ page }) => {
```

Comment:
Please add meaningful test descriptions so it could be clear what they are testing.